tests/unit/src/flixel/FlxObjectTest.hx

```diff
@@ -71,7 +71,65 @@ class FlxObjectTest extends FlxTest
 		object2.velocity.x = -2000;
 		
 		step(60);
-		Assert.isFalse(FlxG.overlap(object1, object2)); 
+		Assert.isFalse(FlxG.overlap(object1, object2));
+	}
+	
+	
+	@Test
```

Comment:
I like to add a comment with the issue number here to give some context (`// #1556`).